packages/cysync-core/src/constants/i18n/en.ts

```diff
@@ -1276,6 +1276,41 @@ const en = {
     close: {
       title: 'Are you sure you want to exit?',
     },
+    cysync: {
+      reset: {
+        confim: {
+          title: 'Are you sure you want to reset the cySync app?',
+          subTitle:
+            'This will erase all your data on your cySync app. Note this will not result in loss of assets',
+        },
+      },
+      releaseNote: {
+        title: 'Release Notes',
+      },
+    },
```

Comment:
```suggestion
      reset: {
        confim: {
          title: 'Are you sure you want to reset the cySync app?',
          subTitle:
            'This will erase all your data on your cySync app. Note this will not result in loss of assets',
        },
      },
      releaseNote: {
        title: 'Release Notes',
      },
```

Reply:
applied